controllers/authController.js

```diff
@@ -11,84 +11,112 @@ const User = db.users;
 
 //CREATE FUNCTION THAT HANDLES TOKEN RESPONSE & COOKIE RESPONSE
 const createSendToken = async (user, statusCode, res) => {
-  // create jwt token with model instance
-  const token = await user.createJwt();
-  const cookieOptions = {
-    expires: new Date(Date.now() + 1 * 60 * 60 * 1000),
-    httpOnly: true,
-  };
-  if (process.env.NODE_ENV === "production") cookieOptions.secure = true;
-
-  // Send token to client
-  res.cookie("jwt", token, cookieOptions);
-
-  res.status(statusCode).json({
-    status: "Success",
-    data: {
-      user,
-      token,
-    },
-  });
+    // create jwt token with model instance
+    const token = await user.createJwt();
+    const cookieOptions = {
+        expires: new Date(Date.now() + 1 * 60 * 60 * 1000),
+        httpOnly: true,
+    };
+    if (process.env.NODE_ENV === "production") cookieOptions.secure = true;
+
+    // Send token to client
+    res.cookie("jwt", token, cookieOptions);
+
+    res.status(statusCode).json({
+        status: "Success",
+        data: {
+            user,
+            token,
+        },
+    });
 };
 
 const signup = async (req, res) => {
-  // console.log(req.headers)
-  // logger.info(req.body)
-  const { username, email, password, displayName } = req.body;
+    // console.log(req.headers)
+    // logger.info(req.body)
 
-  // // validate input
-  if (!(username && email && password && displayName))
-    throw new AppError("All fields are required", 400);
+    const payload = {};
```

Comment:
please what's this 'payload' object?

Reply:
I left it there by mistake. Will rectify that.

Reply:
alright, sure